tests/web/test_info.py

```diff
@@ -0,0 +1,5 @@
+def test_api_endpoint_shows_basic_info(client):
+    from bigchaindb import version
```

Comment:
@libscott the practice of putting some imports under function scope is based on http://docs.pylonsproject.org/en/latest/community/testing.html